r-package/grf/R/input_utilities.R

```diff
@@ -73,17 +73,22 @@ validate_clusters <- function(clusters, X) {
   clusters
 }
 
-validate_samples_per_cluster <- function(samples.per.cluster, clusters) {
+validate_equalize_cluster_weights <- function(equalize.cluster.weights, clusters, sample.weights) {
```

Comment:
This function should have a comment, and maybe a different name. Now it's a little confusing: It does two things, validation & returns samples.per.cluster.

@jtibshirani what do you think?

Reply:
Our `validate_*` methods already do two things at once, maybe we should have used a term like 'prepare' instead of 'validate'. Since this is fairly consistent with our current approach, I think that it's okay to merge.